crates/torii/indexer/src/engine.rs

```diff
@@ -637,10 +637,7 @@ impl<P: Provider + Send + Sync + std::fmt::Debug + 'static> Engine<P> {
         let mut unique_contracts = HashSet::new();
         // Contract -> Cursor
         for (event_idx, event) in events.iter().enumerate() {
-            // NOTE: erc* processors expect the event_id to be in this format to get
-            // transaction_hash:
-            let event_id =
-                format!("{:#064x}:{:#x}:{:#04x}", block_number, transaction_hash, event_idx);
+            let event_id = format!("{:#x}:{:#04x}", transaction_hash, event_idx);
```

Comment:
We should move those formatting to functions to make it easy to maintain an homogenize, wdyt?

Also, having the expected `event_id` documented somewhere or cross-referenced here, would be great for further works.